### ISSUE TITLE
[WCAG/WAI-ARIA] Accessibility audit for core components, making them meet the full criteria for current standards

### ISSUE THREAD

**Issue by Uriziel01** [adobe/aem-core-wcm-components#2917]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Some current accessibility audit reports have shown that multiple components lack proper ARIA/WCAG compatibility according to the last published web standards. WCAG 2.2 and WAI-ARIA 1.2

**Describe the solution you'd like**
There are already partial measures in most core components to support accessibility web standards so it's a matter of supporting more feature complete and/or latest versions of the mentioned standards.

**Are there alternatives?**

**Documentation**
https://www.w3.org/TR/wai-aria-1.2/

https://www.w3.org/TR/WCAG22/
